docs/blogging.rst

```diff
@@ -201,6 +201,9 @@ General writing suggestions
     | *Here’s all I got:*
     | **Read more than you write, live more than you read.**
 
+.. [3]
+   Note of caution, use only the Grammarly website. Don’t install the browser extension. (Those can let someone snoop into everything you type or do in your browser.)
+
```

Comment:
Add some more text, explain not to put any private text or information on the grammarly website or any third party service.